checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitSubchecker.java

```diff
@@ -20,6 +20,18 @@
  *
  * <p>The {@link NullnessChecker} uses this checker as the target (see {@link
  * InitializationChecker#getTargetCheckerClass()}) for its initialization type system.
+ *
+ * <p>You can use the following {@link SuppressWarnings} prefixes with this checker:
+ *
+ * <ul>
+ *   <li>{@code @SuppressWarnings("nullness")} suppresses warnings for nullness, initialization, and
+ *       keyfor annotations
+ *   <li>{@code @SuppressWarnings("nullnesskeyfor")} suppresses warnings for nullness and keyfor
+ *       annotations only
+ *   <li>{@code @SuppressWarnings("nullnessinit")} suppresses warnings for nullness and
```

Comment:
For the Initialization Checker, it is the full "initialization".
It seems inconsistent to abbreviate it as "init" here.
Would "nullnessinitialization" be better?